src/prefect/orion/api/workers.py

```diff
@@ -342,7 +342,7 @@ async def create_work_pool_queue(
     except sa.exc.IntegrityError:
         raise HTTPException(
             status_code=status.HTTP_409_CONFLICT,
-            detail="A worker with this name already exists.",
+            detail="A work pool queue with this name already exists.",
```

Comment:
Are we dead set on these being `work_pool_queues`? I feel like this wastes a bit of an opportunity to overload `work_queue`. 
Something like like `f"A work queue with this name already exists for work pool {work_pool_name!r}"` might allow us to have all of the changes without introducing a new concept of a `work_pool_queue`.

Reply:
That makes sense to me! I updated the message to match your suggestion.